src/driver/collection.ts

```diff
@@ -123,61 +177,119 @@ export class Collection extends MongooseCollection {
         return promise;
     }
 
+    /**
+     * Update a single document in a collection that matches the given filter.
+     * @param filter
+     * @param update
+     * @param options
+     */
     updateOne(filter: Record<string, any>, update: Record<string, any>, options?: UpdateOneOptions) {
         if (options != null) {
             processSortOption(options);
         }
         return this.collection.updateOne(filter, update, options);
     }
 
+    /**
+     * Update multiple documents in a collection that match the given filter.
+     * @param filter
+     * @param update
+     * @param options
+     */
     updateMany(filter: Record<string, any>, update: Record<string, any>, options?: UpdateManyOptions) {
         return this.collection.updateMany(filter, update, options);
     }
 
+    /**
+     * Bulk write not supported.
+     * @param _ops
+     * @param _options
```

Comment:
Let's leave this as `@param ops` and `@param options`, no `_` necessary. I named it `_options` because that's one way to get ESLint to not complain about unused vars, but it looks like there's another way to achieve the same result: https://stackoverflow.com/questions/72583814/how-to-specify-the-jsdoc-rule-for-eslint-to-recognize-unused-parameters . I'll put in a PR later to try that out.

Reply:
Got it, changed all of them in [81c072a](https://github.com/stargate/stargate-mongoose/pull/140/commits/81c072ac15a105f85c0c4fa51da4dffc1f282eda).